### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: events/comments/views.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import json
+
+# Create your views here.
+
+
+class CommentView(View):
+    def get(self, request, id):
+        if not request.user.is_authenticated:
+            return HttpResponse('Permission denied', status=403)
+        comments = Comment.objects.filter(event_id=id)
```
Comment (VKotsovskyi) [inline events/comments/views.py]: It sounds logical to order this by date.

Comment (ghost) [reply events/comments/views.py]: Did it

File: events/comments/views.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        data["author"] = User.get_by_id(request.user.id)
+        if id:
+            data["event"] = Event.get_by_id(id)
+        else:
+            data["parrent_comment"] = Comment.get_by_id(body_requst["parrent_id"])
+        form = CommentForm(data)
+        if not form.is_valid():
+            return HttpResponse(form.errors.as_json(), content_type="text/json", status=400)
+        Comment.objects.create(**data)
+        return HttpResponse('Ok')
```
Comment (yevgenbehen) [inline events/comments/views.py:34]: What about adding status code here?

Comment (ghost) [reply events/comments/views.py:34]: Default status HttpResponse-200.

Comment (yevgenbehen) [reply events/comments/views.py:34]: Okay

**Review by yevgenbehen — APPROVED** [review]
+1

File: events/comments/forms.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from django import forms
+from .models import Comment
+
+class CommentForm(forms.ModelForm):
+    class Meta:
+        model = Comment
+        fields = ["text"]
```
Comment (BorysYudin) [inline events/comments/forms.py]: Please add empty line

File: events/comments/models.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            "id": self.author.id,
+            "name": "{} {}".format(self.author.first_name, self.author.last_name)
+        }
+        return {
+            "id": self.id,
+            "children": children,
+            "text": self.text,
+            "author": author,
+            "date": self.date_add.timestamp()
+        }
```
Comment (BorysYudin) [inline events/comments/models.py]: Please add empty line

File: events/comments/urls.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+from django.conf.urls import url, include
+from . import views
+
+
+urlpatterns = [
+    url(r"^((?P<id>\d+)/)?$", views.CommentView.as_view(), name="comments"),
+]
```
Comment (BorysYudin) [inline events/comments/urls.py]: Please add empty line

**Review by BorysYudin — APPROVED** [review]

File: events/comments/forms.py (hunk 3 above)
Comment (ghost) [reply events/comments/forms.py]: Fix

File: events/comments/models.py (hunk 4 above)
Comment (ghost) [reply events/comments/models.py]: Fix

File: events/comments/urls.py (hunk 5 above)
Comment (ghost) [reply events/comments/urls.py]: Fix

**Pull request merged**